packages/flutter/test/widgets/overscroll_indicator_test.dart

```diff
@@ -362,6 +362,53 @@ void main() {
     expect(painter, paints..save()..translate(y: 0.0)..scale()..circle());
   });
 
+  testWidgets('CustomScrollView overscroll indicator works if there is sliver before center and modify glow position', (WidgetTester tester) async {
+    final Key centerKey = UniqueKey();
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: ScrollConfiguration(
+          behavior: TestScrollBehavior2(),
+          child: NotificationListener<OverscrollIndicatorNotification>(
+            onNotification: (OverscrollIndicatorNotification notification) {
+              if (notification.leading) {
+                notification.paintOffset = 50.0;
+              }
+              return false;
+            },
+            child: CustomScrollView(
+              center: centerKey,
+              physics: const AlwaysScrollableScrollPhysics(),
+              slivers: <Widget>[
+                SliverList(
+                  delegate: SliverChildBuilderDelegate(
+                        (BuildContext context, int index) => Text('First sliver $index',),
+                    childCount: 2,
+                  ),
+                ),
+                SliverList(
+                  key: centerKey,
+                  delegate: SliverChildBuilderDelegate(
+                        (BuildContext context, int index) => Text('Second sliver $index',),
+                    childCount: 5,
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+
+    expect(find.text('First sliver 1'), findsNothing);
+
+    await slowDrag(tester, const Offset(200.0, 200.0), const Offset(0.0, 300.0));
+    expect(find.text('First sliver 1'), findsOneWidget);
+    final RenderObject painter = tester.renderObject(find.byType(CustomPaint));
+    // The painter should follow the scroll direction.
```

Comment:
I thought we are verifying the paint offset is correctly applied, what does the scroll direction do here?

Reply:
Sorry for the comment is not clear, I will fix it.
Line 375 set the `paintOffset` 50.0, `[OverscrollIndicatorNotification.paintOffset]` can special the indicator offset, also see https://api.flutter.dev/flutter/widgets/OverscrollIndicatorNotification/paintOffset.html
So in this test case, the indicator should paint at 50.0 from its edge.